source/install/cloud-changelog.md

```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-03-30](#release-2022-03-30)
 - [Release 2022-03-16](#release-2022-03-16)
 - [Release 2022-03-08](#release-2022-03-08)
 - [Release 2022-03-02](#release-2022-03-02)
 - [Release 2022-02-16](#release-2022-02-16)
 - [Release 2022-02-10](#release-2022-02-10)
-- [Release 2022-02-03](#release-2022-02-03)
+
+## Release 2022-03-30
+
+### Improvements
+
+#### User Interface (UI)
+ - A new field was added to the client configuration to let clients know the database schema version of the server. The applied database migrations have also been added to the **System Console**.
+ - Logs from third-party libraries are now included in default logging configuration.
+ - Improved the license upload flow.
+ - Added performance metrics related to plugin loading on page load.
+ - Changed the Mattermost indigo theme to match the dark theme in code blocks.
+ - Added ``threadAutoFollow`` config setting to the **System Console**.
+ - The start menu start-trial CTA presents a modal exposing the benefits the client gets by starting the trial, encouraging admins to request a trial license and engage them with the product.
```

Comment:
Are the words "start menu" needed here? Would the sentence be accurate without it? Is it a start trial call to action?